SRPTests/RCLinkTests.cs

```diff
@@ -0,0 +1,113 @@
+using Markdig;
+using NUnit.Framework;

```

Comment:
Out of curiosity why did you choose NUnit over MSTest?

Reply:
I seemed to remember we've used NUnit in the past.  I have no preference; happy to switch over the MSTest if you'd prefer.